_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "NewPipe 0.20.7 released: YouTube Mix support, bug fixes and many improvements"
+short: "NewPipe 0.20.7 released"
+date: 2020-12-24 19:00:00
+categories: [pinned, release]
+author: Poolitzer, Opusforlife
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Merry Christmas, if you celebrate that, and almost Happy New Year, if you use a calendar at least close to the Gregorian one. We want to present you a little gift as a good way to start 2021 AD/CE (or the year 12021 in the [Human Era](https://en.wikipedia.org/wiki/Holocene_calendar), according to Kurzgesagt): The next release of NewPipe. It comes with a lot of improvements, fixes, and even a big new feature. So let us dive into this release post!
+
+<!-- more -->
+
+## New in this version
+YouTube Mixes are a thing now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in just the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listening to never-ending playlists and thank [@XiangRongLin](https://github.com/XiangRongLin) for this feature, as well as [@Stypox](https://github.com/Stypox) for reviewing it thoroughly.
+
+This release includes support for another feature from YouTube: video metadata. The little boxes that appear sometimes - telling you that a channel belongs to a public broadcaster, or giving you further information regarding specific search queries - appear in NewPipe as well now. If you do not want to see them, head over to <code><nobr><img src="{{ site.baseurl }}/img/icons/baseline-settings-20px.svg" /> Settings</nobr> > <nobr><img src="{{ site.baseurl }}/img/icons/baseline-language-20px.svg" /> Content</nobr> > <nobr>Show meta info</nobr></code> and disable the toggle there. If you enjoy these boxes however, thank [@TobiGr](https://github.com/TobiGr) for baking them into this release.
```

Comment:
```suggestion
This release includes support for another feature from YouTube: video metadata. The little boxes that appear sometimes - telling you that a channel belongs to a public broadcaster, or giving you further information regarding specific search queries - appear in NewPipe as well now. If you do not want to see them, head over to <code><nobr><img src="{{ site.baseurl }}/img/icons/baseline-settings-20px.svg" /> Settings</nobr> > <nobr><img src="{{ site.baseurl }}/img/icons/baseline-language-20px.svg" /> Content</nobr> > <nobr>Show meta info</nobr></code> and disable the toggle there. If you enjoy these boxes however, thank [@TobiGr](https://github.com/TobiGr) for bringing them into this release.
```

more clear for non coders

Reply:
Baking is not a coding term. xD

But it's fine with me either way.

Reply:
I am fine with both. If you prefer bringing, we can use that.